docs/guide/dev/integration/angular/README.md

```diff
@@ -0,0 +1,234 @@
+
+### `config`
+
+Custom configuration can be passed to the editor via `config` attribute passed to `<ckeditor>` in component template. The following example shows {@link guide/dev/features/toolbar/README how to change the contents of the toolbar}:
+
+```html
+<ckeditor
+	data="<p>Editor' content</p>"
+	[config]="{ toolbar: [ [ 'Bold' ] ] }"
+></ckeditor>
+```
+
+{@linkapi CKEDITOR.config All configuration} options can be changed this way.
+
+### `data`
+
+The initial data of the editor. It can be a static value:
+
+```html
+<ckeditor data="<p>Hello, world!</p>"></ckeditor>
+```
+
+or a shared parent component’s property
+
+```typescript
+@Component( {
+    …
+} )
+export class MyComponent {
+    public editorData = '<p>Hello, world!</p>';
+}
+```
+
+```html
+<ckeditor [data]="editorData"></ckeditor>
+```
+
+### `tagName`
+
+Specifies the tag name of the HTML element on which the editor will be created.
+
+The default tag is `<textArea>`.
+``` html
+<ckeditor tagName="textarea"></ckeditor>
+```
+
+### `readOnly`
+
+Editor can be set to {@link guide/dev/features/readonly/README read-only mode} via `readOnly` property:
+
+```html
+<ckeditor
+	data="<p>Editor's content</p>"
+	readOnly="true"
+><ckeditor>
+```
+
+This property takes precedence over {@linkapi CKEDITOR.config#readOnly `config.readOnly`} setting.
+
+## Supported `@Output` properties
+
+The following `@Output` properties are supported by CKEditor 4 Angular component:
+
+### `ready`
+
+Fires when the editor is ready. It corresponds with the {@linkapi CKEDITOR.editor#instanceReady `editor#instanceReady`} event.
+
+### `change`
+
+Fires when the content of the editor has changed. It corresponds with the {@linkapi CKEDITOR.editor#change `editor#change`} event. For performance reasons this event may be called even when data didn't really changed.
+
+```html
+<ckeditor (change)="onChange($event)"></ckeditor>
+```
+
+``` typescript
+
+import { CKEditor4 } from "@ckeditor/ckeditor4-angular/ckeditor";
+
+@Component( {
+    …
+} )
+export class MyComponent {
+    public onChange( event: CKEditor4.EventInfo ) {
+        console.log( event.editor.getData() );
+    }
+    …
+}
+```
+
+### `dataChange`
+
+Fires when the content of the editor has changed. In contrast to `change` - only emits when data really changed thus can be successfully used with `[data]` and two way `[(data)]` binding.
+
+### `focus`
+
+Fires when the editor's editable is focused. It corresponds with the {@linkapi CKEDITOR.editor#focus `editor#focus`} event.
+
+### `blur`
+
+Fires when the editing view of the editor is blurred. It corresponds with the {@linkapi CKEDITOR.editor.blur `editor#blur`} event.
+
+### Note
+
+Callbacks bound to each of events, are called with {@linkapi CKEDITOR.eventInfo `eventInfo`}. For importing and usage of `EventInfo` interface see [**`change`** example](#change).
```

Comment:
Not each event, as [`dataChange` is fired with a different signature](https://github.com/cksource/ckeditor4-angular/blob/3bfa435/src/ckeditor/ckeditor.component.ts#L293) it gets a plain string.

Reply:
I think it should be changed in the upstream, that is `ckeditor4-angular` integration.